macros/dmt_get_test_sql.sql

```diff
@@ -13,8 +14,17 @@
 
     {% if execute %}
         {# inside an execute block because graph nodes aren't well-defined during parsing #}
-        {% set graph_model = graph.nodes["model." + project_name + "." + model.name] %}
-        {% set ns.test_sql = graph_model.raw_sql %}
+        {% set ns.graph_model = graph.nodes.get("model." + project_name + "." + model.name) %}
+        {# if the model uses an alias, the above call was unsuccessful, so loop through the graph to grab it by the alias instead #}
+        {% if ns.graph_model is none %}
+            {% for node in graph.nodes.values() %}
+                {% if node.alias == model.name %}

```

Comment:
I think we need to be careful here, since there can be multiple models with the same alias as long as they are in different schemas :) so maybe check the schema as well? Or any other ideas how to make sure it picks the correct one?

Reply:
Ah, great catch, thanks. Checking the schema should do it. Fixed!